src/sage/geometry/hyperplane_arrangement/arrangement.py

```diff
@@ -499,7 +499,11 @@ def hyperplanes(self):
 
         OUTPUT:
 
-        An integer.
+        A tuple
+
+        REMARK:
+
+        It applies also to ordered arrangements..
```

Comment:
This remark doesn't make sense here. If it is for a subclass, it should also follow the same paradigm. I would remove this. This applies in other places as well.